clang/lib/CIR/CodeGen/CIRGenCall.cpp

```diff
@@ -76,6 +83,48 @@ static void appendParameterTypes(const CIRGenTypes &cgt,
   cgt.getCGModule().errorNYI("appendParameterTypes: hasExtParameterInfos");
 }
 
+const CIRGenFunctionInfo &
+CIRGenTypes::arrangeCXXStructorDeclaration(GlobalDecl gd) {
+  auto *md = cast<CXXMethodDecl>(gd.getDecl());
+
+  llvm::SmallVector<CanQualType, 16> argTypes;
+  argTypes.push_back(deriveThisType(md->getParent(), md));
+
+  bool passParams = true;
```

Comment:
I see this is checked, but never set to false?

Reply:
Ah, hrmph... Can we set it there ANYWAY, despite doing an NYI?

Reply:
It will be set to the result of a function call in the final implementation. I can add a line to just set it to false there as a place holder if you think that's useful. Same for the other similar case.